src/main/java/io/wispforest/owo/shader/GlProgram.java

```diff
@@ -84,13 +90,7 @@ protected void setup() {}
     }
 
     @ApiStatus.Internal
-    public static void forEachProgram(Consumer<Pair<Function<ResourceFactory, ShaderProgram>, Consumer<ShaderProgram>>> loader) {
+    public static void forEachProgram(Consumer<Runnable> loader) {
         REGISTERED_PROGRAMS.forEach(loader);
     }
```

Comment:
why does this take a `Consumer<Runnable>`? like the only logical action is to run all programs' loaders, so why even pass in a Consumer, just make the method not take in anything and run all the loaders itself

Reply:
I guess